jina/peapods/zmq/__init__.py

```diff
@@ -39,7 +39,7 @@ def __init__(
         self,
         args: 'argparse.Namespace',
         logger: 'JinaLogger' = None,
```

Comment:
here missing an Optional but I think these defaults are wrong

Reply:
I added the default annotation to have it at least correct on syntactical level